src/Umbraco.Web.UI.NetCore/Program.cs

```diff
@@ -14,7 +15,9 @@ public static void Main(string[] args)
         }
 
         public static IHostBuilder CreateHostBuilder(string[] args) =>
-            Host.CreateDefaultBuilder(args)
-                .ConfigureWebHostDefaults(webBuilder => { webBuilder.UseStartup<Startup>(); });
+            Host.CreateDefaultBuilder(args)                
+                .ConfigureWebHostDefaults(webBuilder => { webBuilder.UseStartup<Startup>(); })
+                .UseUmbraco()
+                .UseSerilog();
```

Comment:
I think we should do the `UseSerilog` inside `UseUmbraco`..

Reply:
How do you mean by this? This is Serilog's own setup requirement, see https://nblumhardt.com/2019/10/serilog-in-aspnetcore-3/ and https://github.com/serilog/serilog-aspnetcore

Reply:
By calling `builder.UseSerilog` in `HostBuilderExtensions.UseUmbraco` before returning.. 

Serilog is a dependency due to Umbraco, so I think it is better we set it up, other then require all solutions to do the same

Reply:
Right i see. OK i'll remove this for now and create a new task for that.